tools/setupDNS.sh

```diff
@@ -1,29 +1,35 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-if [ "$(uname)" == "linux-gnu" ]; then
-    echo "\n\nRunning on Linux "
-fi
-
-if [ "$(uname)" == "Darwin" ]; then
-    echo "\n\nRunning on OSX, checking if dependencies are met."
+if [[ "$(uname)" == "Darwin" ]]; then
+    # Check if dnsmasq is already installed.
+    if [[ -n "$(command -v dnsmasq)" ]]; then
+        # The package is installed we do nothing
+        echo "Dnsmasq already installed, stopping daemon and adding config lines"
 
-    if brew ls --versions dnsmasq; then
-      # The package is installed we do nothing
-      echo "\n\nYes DNSMASQ installed."
-
-      # Stop DNSMASQ service to make sure we can continue with the steps below
-      sudo launchctl stop homebrew.mxcl.dnsmasq
+        # Stop DNSMASQ service to make sure we can continue with the steps below
+        sudo launchctl stop homebrew.mxcl.dnsmasq
     else
-      # We know nothing Jon Snow (so we install dnsmasq)
+        # We know nothing Jon Snow (so we install dnsmasq)
         source brew.sh
     fi
 
-    echo 'address=/.sprd/127.0.0.1' > $(brew --prefix)/etc/dnsmasq.conf
+    # Add .yad and .sprd to the config
+    echo 'address=/.yad/127.0.0.1' > $(brew --prefix)/etc/dnsmasq.conf
```

Comment:
`.yad` nog steeds nodig?

Reply:
Niet persee, maar dit was minder werk dan al mijn configs aanpassen. En backwards compat is altijd fijn.